lms/djangoapps/discussion/rest_api/api.py

```diff
@@ -308,6 +308,7 @@ def _format_datetime(dt):
             FORUM_ROLE_MODERATOR,
             FORUM_ROLE_COMMUNITY_TA,
         }),
+        "is_user_admin": request.user.is_staff,
```

Comment:
This is the only condition for IsAdminUser permission?

Reply:
Yes,
here's a reference from docs: https://www.django-rest-framework.org/api-guide/permissions/#isadminuser